modular_skyrat/modules/junior_officer/junior_officer.dm

```diff
@@ -53,7 +53,14 @@
 
 	suit = /obj/item/clothing/suit/armor/vest/peacekeeper/brit
 
-	backpack_contents = list(/obj/item/clothing/gloves/tackler/offbrand, /obj/item/storage/box/gunset/pepperball, /obj/item/restraints/handcuffs/cable = 2, /obj/item/modular_computer/pda/security)
+	backpack_contents = list(
+		/obj/item/clothing/gloves/tackler/offbrand,
+		/obj/item/storage/box/gunset/pepperball,
+		/obj/item/restraints/handcuffs/cable = 2,
+		/obj/item/modular_computer/pda/security,
+		/obj/item/paper = 2,
```

Comment:
Two single sheets of paper?
Why not just give them a clipboard or a full paper sheaf